src/main/java/no/sikt/sws/WorkspaceIndexHandler.java

```diff
@@ -34,15 +34,19 @@ protected IndexResponse processInput(String body, RequestInfo request, Context c
         try {
             var url = workspace + "-" + index;
             logger.info("URL: " + url);
-            var response = openSearchClient.sendRequest(httpMethod, url, body);
+
+            var response = openSearchClient.sendRequest(
+                    httpMethod,
+                    url,
+                    RequestUtil.addWorkspace(body, workspace, index));
             logger.info("response-code:" + response.getStatus());
-            logger.info("response-body:" + response.getBody());
```

Comment:
Can you readd this line? Its usefull for debugging right now to check what the output is before you strip it

Reply:
You can do whatever you want with it, read output before stripping or/and after. I really don't have an opinion about it, put it where ever you want it.